examples/talk-llama/talk-llama.cpp

```diff
@@ -370,13 +353,32 @@ int main(int argc, char ** argv) {
                 return 1;
             }
             session_tokens.resize(n_token_count_out);
+            for (size_t i = 0; i < session_tokens.size(); i++) {
+                embd_inp[i] = session_tokens[i];
+            }
```

Comment:
Isn't this overwriting on top of the prompt tokenized on line 339?
Is this what we want?

Reply:
@ggerganov with this approach I'm replacing the default prompt: https://github.com/ggerganov/whisper.cpp/blob/b8064208733a3d9f5a9f418448ec46a39333e272/examples/talk-llama/talk-llama.cpp#L205
And instead I'm loading the prompts from the session, so it will resume the context and keep the consistency of the previous conversations